hack/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.7.3
+RUN curl https://glide.sh/get | sh
```

Comment:
I got that from the glide docs, but if there's a better way i'm ok with that.
btw - this isn't root on your machine, its just root in the container - so its safe.